randovania/gui/lib/common_qt_lib.py

```diff
@@ -294,3 +296,9 @@ def set_icon_data_paths(label: QtWidgets.QLabel):
     repl = rf'<img src="{get_data_path().as_posix()}/\g<1>"/>'
     new_text = image_pattern.sub(repl, label.text())
     label.setText(new_text)
+
+
+def alert_user(parent: QtWidgets, options: Options):
+    QtWidgets.QApplication.alert(parent)
+    if not parent._options.disable_sound_alerts:
```

Comment:
The intent of the `options` argument is that you don't need to find it in an undocumented place in the widget

Reply:
I'm unsure what that means

Reply:
a `QtWidget` having a `_options` field is not a rule, so you shouldn't be doing `parent._options`. But this is exactly why I suggest for this function to receive an `Options` as argument, so you can just use that directly

Reply:
Oh, I'm silly - I didn't realize that I still was calling on parent._options instead of the passed in options

Reply:
Fixed!